### DISCUSSION TITLE
Multiple final destinations

### DISCUSSION THREAD

**Discussion by ChrisFirgaira** [madMAx43v3r/chia-plotter#596] [General]
Hi All,

I apologise if this has already been discussed I wasn't able to search/find an answer

Is there a way to define multiple destinations? i.e. fill destination drive1, then 2 then 3 etc

**Comment by SebMoore** [comment]
See #60. Answer is no at the moment unfortunately.

**Comment by ebeng** [comment]
> Hi All,
> 
> 
> 
> I apologise if this has already been discussed I wasn't able to search/find an answer
> 
> 
> 
> Is there a way to define multiple destinations? i.e. fill destination drive1, then 2 then 3 etc

Searching for this one as well.
I know that madmax is the way forward for plotting, but untill Plotman or some such is not out there, I cant loadbalance all the plots to several NFS shares at this moment. 
Moving around with the plots is also not ideal, as this can impact also the performance when plots are being created. Maybe that I will create a few more extra plots with madmax, but not for the long run probably untill some plotmanager is available.

**Comment by Kiggles** [comment]
You can still run plotman using the archiving features without it managing anything about plotting.

For Linux users with many many drives on a single device, as an alternative, look into MergerFS. Then just point your destination to the MergerFS directory and let it handle the load balancing, even if you have multiple machines copying over network to the same MergerFS destination dir.

**Reply by ebeng** [reply]
Will surely look into this one.

**Comment by Dartellum** [comment]
I wrote a bash script that finds a "drive" with free space to write the plot, logs, and loops through as many times as you want to run: https://github.com/Dartellum/chia-madmax-plot_loop.

**Comment by slickdakine** [comment]
Mergerfs works great for that:
https://github.com/trapexit/mergerfs

**Comment by number435398** [comment]
I've also requested this https://github.com/madMAx43v3r/chia-plotter/issues/547
